src/setup/txt_joyaxis.c

```diff
@@ -150,13 +150,14 @@ static boolean CalibrateAxis(txt_joystick_axis_t *joystick_axis)
 
     for (i = 0; i < SDL_JoystickNumAxes(joystick_axis->joystick); ++i)
     {
-       //if (bad_axis[i])
-       //{
-       //    continue;
-       //}
-
         axis_value = SDL_JoystickGetAxis(joystick_axis->joystick, i);
 
+        if (joystick_axis->bad_axis[i])
+        {
+            printf("CalibrateAxis() Ignoring bad axis #%i\n", i);
```

Comment:
Please remove this `printf()` message - there is already a similar message further down in the file.